apps/src/storage/dataBrowser/DataLibraryPane.jsx

```diff
@@ -35,14 +36,24 @@ class DataLibraryPane extends React.Component {
   };
 
   importTable = datasetInfo => {
-    if (datasetInfo.current) {
-      FirebaseStorage.addCurrentTableToProject(
-        datasetInfo.name,
-        () => {},
-        this.onError
-      );
+    // TODO: post-firebase-cleanup, remove this conditional: #56994
+    if (isFirebaseStorage()) {
+      if (datasetInfo.current) {
+        storageBackend().addCurrentTableToProject(
+          datasetInfo.name,
+          () => {},
+          this.onError
+        );
+      } else {
+        storageBackend().copyStaticTable(
+          datasetInfo.name,
+          () => {},
+          this.onError
+        );
+      }
     } else {
-      FirebaseStorage.copyStaticTable(datasetInfo.name, () => {}, this.onError);
+      storageBackend().addSharedTable(datasetInfo.name);
+      refreshCurrentDataView();
```

Comment:
In datablock storage the difference between current and static tables is collapsed, see https://github.com/code-dot-org/code-dot-org/pull/56279/files#r1524559478